File: src/main/java/com/love/guess/GameFor.java
package com.love.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        int secret = new Random().nextInt(10) + 1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        for(int i=1;i<=4;i++) {
            System.out.print("please enter a number ("+i+"/4) :\t");
            int number = scanner.nextInt();
            System.out.println("第"+i+"次:\t" + number);
            if(number>secret){
                System.out.println("lower");
            }else if(number<secret){
                System.out.println("higher");
            }else {
                System.out.println("Great,the number is "+number);
                break;
            }
        }
        System.out.println("結束");
    }

}

File: src/main/kotlin/com/love/GameFor.kt
package com.love

import java.util.*

fun main(args: Array<String>) {
    val secret=Random().nextInt(10)+1
    println(secret)
    val scanner=Scanner(System.`in`)
    for (i in 1..4) {
        print("please enter a number (${i}/4) :\t")
        val number = scanner.nextInt()
        println("第${i}次:\t$number")
        if(number>secret){
            println("lower")
        }else if(number<secret){
            println("higher")
        }else{
            println("Great,the number is\t$number")
            break
        }
    }
    println("結束")
}